controller/uninstall_controller.go

```diff
@@ -491,50 +508,62 @@ func (c *UninstallController) deleteInstanceManagers(instanceManagers map[string
 				if err = c.ds.DeletePod(pod.Name); err != nil {
 					return
 				}
-				logrus.WithFields(logFields).Infof("Removed instance manager")
+				log.Info("Removed instance manager")
 			}
 
 			if err = c.ds.RemoveFinalizerForInstanceManager(im); err != nil {
 				err = errors.Wrapf(err, "Failed to remove finalizer")
 				return
 			}
-			logrus.WithFields(logFields).Infof("Removed finalizer")
+			log.Info("Removed finalizer")
 		}
 	}
 	return
 }
 
 func (c *UninstallController) deleteManager() (bool, error) {
+	log := c.logger.WithFields(
```

Comment:
For the DaemonSet/ReplicaSet you can add the getLoggerForX functions to this controller for now.
the idea is that the fieldName is `daemonSet`, with the value of the daemonsSet name.